cdap-docs/developers-manual/source/building-blocks/datasets/system-custom.rst

```diff
@@ -101,7 +101,14 @@ To create a Dataset of type ``UniqueCountTable``, add the following into the App
 
 You can also pass ``DatasetProperties`` as a third parameter to the ``createDataset`` method.
 These properties will be used by embedded Datasets during creation and will be available via the
-``DatasetSpecification`` passed to the Dataset constructor.
+``DatasetSpecification`` passed to the Dataset constructor. For example, to create a 
```

Comment:
I don't think there is. If there is, I can't easily find it. Here is a list of current constants:
http://docs.cask.co/cdap/current/en/reference-manual/javadocs/constant-values.html
and in 2.7.0:
http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-SNAPSHOT/en/reference-manual/javadocs/constant-values.html